.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster.py

```diff
@@ -70,6 +71,20 @@ def build_repo_wide_ruff_steps() -> List[CommandStep]:
     ]
 
 
+def build_repo_wide_prettier_steps() -> List[CommandStep]:
+    return [
+        CommandStepBuilder(":prettier: prettier")
+        .run(
+            "pushd js_modules/dagster-ui/packages/eslint-config",
+            "yarn install",
```

Comment:
I believe this uses the built in node 12 instead of node 16, which we install to use for the dagit steps. I think we can get away without it since we're just doing an install, but we might need to copy https://github.com/dagster-io/dagster/blob/johann/11-15-add_prettier_to_bk/.buildkite/dagster-buildkite/dagster_buildkite/steps/dagster_ui.py#L33 at some point